File: aulas/aula002/app/src/main/java/aluno/gabriel/aula002/view/SplashActivity.java
package aluno.gabriel.aula002.view;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;

import aluno.gabriel.aula002.R;
import aluno.gabriel.aula002.model.Cliente;

public class SplashActivity extends AppCompatActivity {

    Cliente cliente;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);

        trocarTela();
    }

    protected void trocarTela(){
        boolean Handler = new Handler().postDelayed(new Runnable() {
            @Override
            public void run() {
                cliente = new Cliente(
                        "Hugo Tevéz Holambra Diniz",
                        "Masculino",
                        35,
                        "huguitotevez28@hotmail.com",
                        "+54 159324-6808"
                );
                Intent intent = new Intent(SplashActivity.this, MainActivity.class);
                Bundle bundle = new Bundle();
                bundle.putString("nome", cliente.getNome());
                bundle.putString("sexo", cliente.getSexo());
                bundle.putInt("idade", cliente.getIdade());
                bundle.putString("email", cliente.getEmail());
                bundle.putString("telefone", cliente.getTelefone());
                intent.putExtras(bundle);
                startActivity(intent);
                finish();

            }
        },5000);
    }
}

File: treinos/treino015/app/src/main/java/aluno/gabriel/treino015/view/SplashActivity.kt
package aluno.gabriel.treino015.view

import aluno.gabriel.treino015.R
import android.content.Intent
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.os.Handler
import android.os.Looper

class SplashActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)

        trocarTela()
    }

    fun trocarTela(){
        Handler(Looper.getMainLooper()).postDelayed({
            startActivity(Intent(this@SplashActivity, MainActivity::class.java))
            finish()
        },3000)
    }
}